SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -57,27 +48,25 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal
 
 * **Configuration**
 
-    * **Project Root Folder**</br>
-    If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<ProjectFileLocation>/../spatial`.
+    * **Project root folder**</br>
+    If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<GameRoot>/../spatial`.
 
-    * **Launch Configuration**</br>
+    * **Launch configuration**</br>
     The [launch configuration file](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config) (SpatialOS documentation) to use when running `spatial local launch` using the `Launch` button.
```

Comment:
We usually put the note that it's outside docs inside the link.
Also, may be better to keep the link in the GDK glossary.